drivers/filesystems/fastfat/create.c

```diff
@@ -791,6 +791,11 @@ VfatCreateFile(
                 Attributes |= FILE_ATTRIBUTE_ARCHIVE;
             }
             vfatSplitPathName(&PathNameU, NULL, &FileNameU);
+
+            if ((FileNameU.Length == sizeof(WCHAR) && FileNameU.Buffer[0] == L'.') ||
```

Comment:
It would be nice to also use the function here. You could put it in `string.c` to make it clear it's a global utility.
There is also another place (`VfatOpenFile`) where it could make sense to use this function.

Reply:
@ThFabba, Which string.c? There are many in the code. Thanks.

Reply:
`drivers/filesystems/fastfat/string.c` near `vfatIsLongIllegal` (with a declaration in vfat.h, also near `vfatIsLongIllegal`)

Reply:
Got it. Thanks.

Reply:
Done. I did not add to VfatOpenFile yet. I will review. Thanks.

Reply:
OK. I have used the IsDotOrDotDot inside VfatOpenFile now.